04-data-tree/index.js

```diff
@@ -0,0 +1,40 @@
+class Node {
+  constructor(value) {
+    this.value = value;
+    this.children = [];
+  }
+
+  addChild(child) {
+    this.children.push(child);
+  }
+  find(value) {
```

Comment:
It doesn't check the value of the root node.

Reply:
I dont understand

Reply:
what if the value of the root is the value that you are looking for? (root = the very first node)